clash/bloody_grail.tex

```diff
@@ -0,0 +1,116 @@
+% !TeX spellcheck = en_US
+\addscenariosection{1}{Clash Scenario}{Bloody Grail}{\images/bloody-grail.png}
+
+\begin{multicols*}{2}
+
+\textbf{Author:} Re4XN
+
+\textbf{Source:} \href{https://discord.com/channels/740870068178649108/1239631918643941509}{Archon Studios Discord}
+
+\textit{A Lady clad in dark robes visits you in your dreams, offering you a golden chalice full of blood-red wine. Waking up in a cold sweat, you decide to embark on a quest to find this unholy artifact.}
+\subsection*{\MakeUppercase{Player Count}}
+A scenario for 2 or 4 players.
+\subsection*{\MakeUppercase{Scenario Length}}
+This scenario plays out over 13 rounds.
+\subsection*{\MakeUppercase{Scenario Length}}
+This scenario plays out over 13 rounds.
+\subsection*{\MakeUppercase{Map Setup}}
+Take the following Map tiles and set them up as shown in the scenario map layout:
+
+\textbf{For a 2-player scenario:}
+\begin{itemize}
+    \item 2 × Starting (I) Map tile
+    \item 4 × Near (IV-V) Map tile, all of which must contain an Obelisk
+    \item 4 × Far (II-III) Map tile, 2 of which must contain a Settlement – each player gets 2 of these tiles, 1 of which must contain a Settlement
+    \item 1 × Center (VI-VII) Map tile, which must contain the Grail field
+\end{itemize}
+
+\textbf{For a 4-player scenario:}
+\begin{itemize}
+    \item 4 × Starting (I) Map tile
+    \item 6 × Near (IV-V) Map tile, 4 of which must contain an Obelisk
+    \item 8 × Far (II-III) Map tile, 4 of which must contain a Settlement – each player gets 2 of these tiles, 1 of which must contain a Settlement
+    \item 1 × Center (VI-VII) Map tile, which must contain the Grail field
+\end{itemize}
+\textbf{\MakeUppercase{Note:}} Before placing the Near tiles, separate them into 2 piles (with and without an Obelisk). Put them alternately, so that the tiles with an Obelisk are not placed adjacent to each other.
+\subsection*{\MakeUppercase{Starting Resources}}
+Each player starts with the following resources:
+\begin{center}
+    \textbf{13 ×} \svg{gold}   \textbf{2 ×}   \svg{building_materials}   \textbf{1 ×} \svg{valuables}
+\end{center}
+\subsection*{\MakeUppercase{Setting a Player's Income}}
+Set the income on the income tracker to:
+\begin{center}
+    \textbf{“10”} \svg{gold}   \textbf{“0”}   \svg{building_materials}   \textbf{“0”} \svg{valuables}
+\end{center}
```

Comment:
```suggestion
Set the income on the income tracker to:\par
{\centering\hfill\textbf{“10”} \svg{gold} \hfill \textbf{“0”} \svg{building_materials} \hfill \textbf{“0”} \svg{valuables}\hfill}
```

Reply:
We should keep this consistent and maybe have clash template same as coop

<img width="339" alt="Screenshot 2024-06-26 at 0 27 20" src="https://github.com/qwrtln/Homm3BG-mission-book/assets/1928291/b8ce68c8-e025-4fbc-95dd-ffa5ff5d565b">

Reply:
Same with timed events and such.

Reply:
I'd actually suggest using the same layout from the original books, since that is what players will be familiar with and a new layout might be confusing / unintuitive. But if it is agreed to use a different one, that is fine by me.